apps/arena/lib/arena/game/effect.ex

```diff
@@ -0,0 +1,103 @@
+defmodule Arena.Game.Effect do
+  @moduledoc """
+  This module contains all the functionality related to effects
+  """
+
+  def put_effect(game_state, player_id, owner_id, effect) do
+    last_id = game_state.last_id + 1
+    expires_at = System.monotonic_time(:millisecond) + effect.duration_ms
```

Comment:
not every effect have expiration time, for example the effect that the pool applies doesn't, they're added/deleted when a player collides or stop colliding with a pool, in fact when you try to ult with valtimer this pr breaks because it doesn't have a duration:

```
** (KeyError) key :duration_ms not found in: %{
  name: "singularity",
  remove_on_action: false,
  effect_mechanics: %{
    pull: %{force: 1.0, effect_delay_ms: 0},
    damage: %{damage: 0.01, effect_delay_ms: 400}
  }
}
```

Reply:
Fixed on https://github.com/lambdaclass/mirra_backend/pull/425/commits/c57e497b3d9d1be691023f3b7c325bc0896d92f5